tests/test_monitors.py

```diff
@@ -0,0 +1,184 @@
+from scrapy import Spider
+from scrapy.crawler import Crawler
+from spidermon.contrib.scrapy.runners import SpiderMonitorRunner
+from spidermon.contrib.scrapy.monitors import (
+    FinishReasonMonitor, ItemCountMonitor, LogMonitor, UnwantedHttpStatus,
+    SPIDERMON_MIN_ITEMS, SPIDERMON_EXPECTED_FINISH_REASONS,
+    SPIDERMON_MAX_ERROR, SPIDERMON_MAX_UNWANTED_HTTP_CODES)
+from spidermon import MonitorSuite
+
+
+def new_crawler():
+    return Crawler(Spider)
+
+
+def new_suite(monitors):
+    return MonitorSuite(monitors=monitors)
+
+
+def test_needs_to_configure_item_count_monitor():
+    """ Should raise an exception when ItemCountMonitor it's not configured """
+    crawler = new_crawler()
+    spider = Spider('dummy')
+    suite = new_suite([ItemCountMonitor, ])
+    runner = SpiderMonitorRunner(spider=spider)
+    crawler.stats.set_value('item_scraped_count', 10)
+    data = {
+            'stats': crawler.stats.get_stats(),
+            'crawler': crawler,
+            'spider': spider,
+            'job': None,
+    }
+    runner.run(suite, **data)
+    for r in runner.result.monitor_results:
+        assert('NotConfigured' in r.error)
+
+
+def test_item_count_monitor_should_fail():
```

Comment:
I think the test will be more robust, if you create more scenarios. You can use [pytest.fixtures](https://docs.pytest.org/en/latest/example/parametrize.html?highlight=pytest.fixture#indirect-parametrization-with-multiple-fixtures) to achieve it.

Some scenarios I can imagine for this test:
- number of items equal to SPIDERMON_MIN_ITEMS
- number of items greater than SPIDERMON_MIN_ITEMS
- number of items lower than SPIDERMON_MIN_ITEMS
- SPIDERMON_MIN_ITEMS not a valid integer

Rename the test would be good, if you use the fixtures.